docs/Customization/Firrtl-Transforms.rst

```diff
@@ -0,0 +1,73 @@
+.. _firrtl-transforms:
+
+Adding a Firrtl Transform
+=========================
+
+After writing the Chisel RTL, you can make further modifications by adding transforms during the FIRRTL compilation phase.
+As mentioned in Section <>, transforms are modifications that happen on the FIRRTL IR that can modify a circuit.
+Transforms are a powerful tool to take in the FIRRTL IR that is emitted from Chisel and run analysis (https://www.youtube.com/watch?v=FktjrjRVBoY) or convert the circuit into a new form.
+
+Where to add transforms
+-----------------------
+
+In Chipyard, the FIRRTL compiler is called multiple times to create a "Top" file that has the DUT and a "Harness" file that has all harness collateral.
```

Comment:
> a "Harness" file that has all harness collateral
This is kind of confusing/misleading. There's not one file that has all harness collateral- the verilog, annotations, and .fir are all separate files. If we're talking strictly about verilog, then "collateral" is a bit confusing.
```suggestion
In Chipyard, the FIRRTL compiler is called multiple times to create a "Top" file containing the DUT and a "Harness" file containing the test harness, which instantiates the DUT. The "Harness" file does not contain the DUT's module definition or any of its submodules.
```